docs/app/Examples/Button/Inverted.js

```diff
@@ -0,0 +1,43 @@
+import React, {Component} from 'react';
+import {Button} from 'stardust';
+
+export default class extends Component {
+  render() {
+    return (
+      <div>
+        <div className='ui inverted segment'>
```

Comment:
Let's use Stardust components where ever possible, there is a `<Segment className='inverted'>` that will do this.